CSharpMath.Editor/MathKeyboard.cs

```diff
@@ -25,7 +24,7 @@ public class MathKeyboard<TFont, TGlyph> where TFont : IFont<TGlyph> {
       blinkTimer.Elapsed += (sender, e) => {
         switch (CaretState) {
           case MathKeyboardCaretState.Shown:
-          case MathKeyboardCaretState.ShownThroughPlaceholder:
+            if (LaTeXSettings.PlaceholderBlinks || !(MathList.AtomAt(_insertionIndex) is Atoms.Placeholder _))
             CaretState = MathKeyboardCaretState.TemporarilyHidden;
```

Comment:
Can you fix indentation here?

i.e.

```csharp
if (LaTeXSettings.PlaceholderBlinks || !(MathList.AtomAt(_insertionIndex) is Atoms.Placeholder _))
CaretState = MathKeyboardCaretState.TemporarilyHidden;
```

=>

```cs
if (LaTeXSettings.PlaceholderBlinks || !(MathList.AtomAt(_insertionIndex) is Atoms.Placeholder _))
  CaretState = MathKeyboardCaretState.TemporarilyHidden;
```

Reply:
Fixed in 1db0430. I also switched the order
`if (!(MathList.AtomAt(_insertionIndex) is Atoms.Placeholder _) || LaTeXSettings.PlaceholderBlinks)`